internal/controllers/operator_controller_test.go

```diff
@@ -575,43 +580,6 @@ var _ = Describe("Operator Controller Test", func() {
 				})
 			})
 		})
-		When("the selected bundle's image ref cannot be parsed", func() {
```

Comment:
Removed this test as the bundle path retrieval no longer has an error path with the new struct.